httpd/handler.go

```diff
@@ -164,7 +164,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 	for {
 		if err := dec.Decode(&br); err != nil {
 			if err.Error() == "EOF" {
-				w.WriteHeader(http.StatusOK)
+				w.WriteHeader(http.StatusCreated)
 				return
```

Comment:
Hmmm. Not a big fan of this -- it's way too low-level a detail, and feels it too pedantic. I think the API should return 200 and be done with it -- I'm thinking of what is natural for client programming. elasticsearch returns 200 when indexing documents in its store.